pkg/client/core/api/direct/direct_clients.go

```diff
@@ -0,0 +1,22 @@
+package direct
+
+import "github.com/oslokommune/okctl/pkg/client"
+
+// Clients contains the direct clients
+type Clients struct {
```

Comment:
I suggest calling this struct `Toolchain`. Likely we'll move the actual server side components into this one to remove the abstraction layer later on

Reply:
:+1: